examples/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ set(GTEST_LIBS gtest gtest_main)
 # Examples
 ################################################################################
 set(EXAMPLES_FILES
+  dilate
```

Comment:
This kernel is essentially 2 layers of a WaveNet.  https://arxiv.org/pdf/1609.03499.pdf

While the WaveNet uses dilated convolutions, the dilation part isn't in this kernel specifically, so a name like `wavenet_layer_2` or something similiar might be more appropriate